index.js

```diff
@@ -1,39 +1,45 @@
-if (process.env.NODE_ENV != 'production') {
-  require('dotenv').config({ path: '.env.development.local' });
+if (process.env.NODE_ENV != "production") {
+  require("dotenv").config({ path: ".env.development.local" });
```

Comment:
It's all good.
Also i noticed your linter plugin doesn't use the config file to format. A portion of the changes in the PR are ' to " formats, does the linter not complain when you run the dev server ?